src/App.jsx

```diff
@@ -14,70 +16,103 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const ItemGood = (
+  {
+    good, goodsMod, selectedGood, setselectedGood, goodsCh, setGoodsCh,
+  },
+) => {
```

Comment:
Fix all cases if present

```suggestion
export const ItemGood = ({
    good, 
    goodsMod, 
    selectedGood, 
    setselectedGood, 
    goodsCh, 
    setGoodsCh,
}) => {
```